src/components/Topics.jsx

```diff
@@ -0,0 +1,34 @@
+import { useEffect } from "react";
+import { useState } from "react";
+import { Link } from "@mui/material";
+import fetchTopics from "../controllers/fetchTopics";
+import { Link as RouterLink } from "react-router-dom";
+import { Paper, Typography, Tooltip } from "@mui/material";
+
+function Topics() {
+  const [topics, setTopics] = useState([]);
+  useEffect(() => {
+    const getTopics = async () => {
+      const newTopics = await fetchTopics();
+      setTopics(newTopics);
+    };
+    getTopics();
+  }, []);
+  return (
+    <>
```

Comment:
I'd suggest a <ul> would be a better parent element for your mapped topic list than a react fragment. Please remember to stick support accessibility with semantic html elements

Reply:
> I'd suggest a
> 
> would be a better parent element for your mapped topic list than a react fragment. Please remember to stick support accessibility with semantic html elements

look at what the component is mounted in?